Libraries/pbxbuild/Sources/Tool/CopyResolver.cpp

```diff
@@ -95,6 +100,47 @@ resolve(
     toolContext->invocations().push_back(invocation);
 }
 
+void Tool::CopyResolver::
+resolve(
+    Tool::Context *toolContext,
+    pbxsetting::Environment const &baseEnvironment,
+    std::vector<std::string> const &inputs,
+    std::string const &outputDirectory,
+    std::string const &logMessageTitle) const
+{
+    return ResolveInternal(
+        _tool,
+        toolContext,
+        baseEnvironment,
+        inputs,
+        ext::nullopt,
+        outputDirectory,
+        logMessageTitle);
+}
+
+void Tool::CopyResolver::
+resolve(
+    Tool::Context *toolContext,
+    pbxsetting::Environment const &baseEnvironment,
+    std::vector<Phase::File> const &inputs,
+    std::string const &outputDirectory,
+    std::string const &logMessageTitle) const
+{
+    std::vector<std::string> inputPaths;
+    std::transform(inputs.begin(), inputs.end(), std::back_inserter(inputPaths), [&](Phase::File const &input) -> std::string {
```

Comment:
I think I copied these lines from somewhere else that used this syntax, but not sure why.